internal/config/config.go

```diff
@@ -504,6 +504,28 @@ func DeleteNamespace(name string) error {
 	return util.NewNotFoundError("Could not find namespace " + name)
 }
 
+// RenameNamespace renames a namespace
+func RenameNamespace(name, newname string) error {
+	if name == "" {
+		name = conf.CurrentNamespace
+		if name == conf.DefaultNamespace {
```

Comment:
That if will only trigger if namespace is current namespace.
So, if I explicitly specify `iofogctl rename namespace default my_new_name`, the if will not trigger, and it will let me rename default namespace. Is this what you want ? 
It creates some inconsistency in the behavior